### PR TITLE
JBPM-5590: Stunner - add properties to reusable subprocess

### PR DESCRIPTION
**Pull request by jlindop**

### CODE REVIEW

File: kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-client/src/main/resources/org/kie/workbench/common/stunner/bpmn/client/resources/i18n/StunnerBPMNConstants.properties [hunk 1]
```diff
@@ -24,6 +24,9 @@ BPMNProperties.timeCycleLanguage=Time Cycle Language
 BPMNProperties.timeDate=Time Date
 BPMNProperties.timeDuration=Time Duration
 BPMNProperties.calledElement=Called Element
+BPMNProperties.independent=Independent
+BPMNProperties.waitForCompletion=Wait for Completion
+BPMNProperties.isAsync=Is Async
```
Comment (Rikkola) [inline kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-client/src/main/resources/org/kie/workbench/common/stunner/bpmn/client/resources/i18n/StunnerBPMNConstants.properties]: I see you have these same strings above in `BPMNProperty` classes as plain text. Is there some sort of magic used to pull the right i18n in client side? 
If there is I could reuse that.

Comment (jlindop) [inline kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-client/src/main/resources/org/kie/workbench/common/stunner/bpmn/client/resources/i18n/StunnerBPMNConstants.properties]: I've asked Pere to comment on this.

**Comment by jlindop** [comment]
jenkins please retest this

**Comment by hasys** [comment]
Hi guys. 
@jlindop, @pefernan, @Rikkola, any updates on it?

**Comment by jlindop** [comment]
I've just taken the isAsync property out of this PR as it's being added in https://github.com/droolsjbpm/kie-wb-common/pull/673
@tsurdilo Will you review so it can be merged? Thanks.

**Comment by tsurdilo** [comment]
@jlindop thanks! yes reviewing right now.

**Comment by tsurdilo** [comment]
+1

File: kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-client/src/main/resources/org/kie/workbench/common/stunner/bpmn/client/resources/i18n/StunnerBPMNConstants.properties (hunk 1 above)
Comment (pefernan) [inline kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-client/src/main/resources/org/kie/workbench/common/stunner/bpmn/client/resources/i18n/StunnerBPMNConstants.properties]: @Rikkola I'm late but I'll try to explain.

This .properties file is related only to Field Labels for the properties forms, the others are related to the on BPMNProperty classes are kind of "default literals" to be used inside stunner editor (on the component palette for example).

In a future PR (I'll send it today) the Form Fields label keys and the Stunner models literals will be unified on a same properties field, althought the default literals on classes will still exist for a while.

Makes sense?

**Pull request merged**